components/rating/rating.css

```diff
@@ -0,0 +1,23 @@
+/* @import url('../../components/root.css'); */
+
+
+@import url('../../components/framer.css');
+
+.rating-icon{
+    cursor: pointer;
+    color: gray;
+    cursor: pointer;
+    transition: all 0.2s ease;
+}
+.rating-icon-size{
+    font-size: 40px !important;
```

Comment:
Maybe avoid the use of `!important` as it can create problems down the line. Try to solve the problem using specificity.